packages/app-store-cli/src/components/AppCreateUpdateForm.tsx

```diff
@@ -93,6 +93,7 @@ export const AppForm = ({
         { label: "CRM", value: "crm" },
         { label: "Messaging", value: "messaging" },
         { label: "Payment", value: "payment" },
+        { label: "Audit Log", value: "auditLogs" },
```

Comment:
Need category available to properly categorize apps. If auditLog implementations are not categorizes properly, they will not work